test/Microsoft.DocAsCode.MarkdigEngine.Tests/RowsTest.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.MarkdigEngine.Tests
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Xml;
+
+    using MarkdigEngine.Extensions;
+
+    using Microsoft.DocAsCode.Plugins;
+    using Xunit;
+
+    public class RowTest
+    {
+        private void TestMarkup(string source, string expected)
+        {
+            var marked = TestUtility.MarkupWithoutSourceInfo(source, "Topic.md");
+            var message = source + "\r\n\r\n" + expected;
+            Assert.Equal(expected.Replace("\r\n", "\n"), marked.Html);
+        }
+
+        [Fact]
+        [Trait("Related", "Row")]
+        public void RowTest_SimpleRow()
+        {
+            var source = @":::row:::
+    :::column:::
+        This is where your content goes.
+    :::column-end:::
+:::row-end:::
+";
```

Comment:
It seems that following syntax is also supported, is it by design?
```
:::column:::
    :::row:::
        This is where your content goes.
    :::row-end:::
:::column-end:::
```

Reply:
Yes, you're correct

Reply:
But from the user story, column extension are whitelisted as the only valid content for the row extension. The sample I provide should not be supported, am I correct?

Reply:
@wibjorn I think it should be forbidden.

Reply:
@qinezh @vwxyzh yes, we would like to restrict the content of the row extension, but Will did not find an easy way to does this.  For now we'll defer this as the content authors who will be using these feature is a controlled group.

In the future, this would be great to know but we don't need to do it now.

We do need to get this approved and made available asap as Windows content is blocked without this extension.  How soon can we get this published?

/cc @wibjorn

Reply:
@wibjorn any updates here?

Reply:
@adkinn @wibjorn sure, but please add a `todo` :smile:

Reply:
@adkinn @qinezh @vwxyzh 

Sounds good. I don't have write access, so someone with this access will have to approve the PR.

Reply:
Hi @adkinn could you let us know the expectation of release date? The next release date of DocFX would be 3/16.

/cc @wibjorn

Reply:
@qinezh We'd like to release it as a hotfix if possible. This is an urgent request for Windows content.

Reply:
@adkinn sure, I'll update you once the hotfix is released.

Reply:
@adkinn the hotfix has been released, please help to verify.